Objective-C/CBLBlob.mm

```diff
@@ -121,9 +121,7 @@ - (instancetype) initWithDatabase: (CBLDatabase*)db
     self = [super init];
     if(self) {
         _db = db;
-        NSMutableDictionary *tempProps = [properties mutableCopy];
-        tempProps[kTypeMetaProperty] = nil;
-        _properties = tempProps;
+        _properties = properties;
```

Comment:
It doesn't seem like it's worth to keep the _properties anymore as all of the properties have been read and stored in  variables.